app/main.py

```diff
@@ -1,8 +1,50 @@
+from typing import AnyStr
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int
+                 ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: AnyStr) -> float:
```

Comment:
The `calculate_washing_price` method should take a `Car` object as its parameter, not `AnyStr`. This will cause a runtime error since `AnyStr` does not have the attributes `comfort_class` and `clean_mark`.